data/selinux/snappy.fc

```diff
@@ -21,6 +21,8 @@ HOME_DIR/snap(/.*)?			gen_context(system_u:object_r:snappy_home_t,s0)
 
 
 /usr/bin/snap			--	gen_context(system_u:object_r:snappy_exec_t,s0)
+/usr/bin/snapctl		--	gen_context(system_u:object_r:snappy_exec_t,s0)
+/usr/bin/ubuntu-core-launcher	--	gen_context(system_u:object_r:snappy_exec_t,s0)
```

Comment:
I think we can get rid of this one. It's not needed anymore and will not be shipped in new installs